### PR TITLE
Use directory docs for the documentation

### PR DESCRIPTION
**Pull request by riccardoscalco**
In this PR I moved documentation files in the `docs` directory. Once approved, we need to update Github Pages config.

Moreover, I edited the readme, update the year in the license and add small changes to the `package.json`.

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2018 Nextbit
```
Comment (rpadovani) [inline LICENSE]: I think you should do `2018-2019`

Comment (riccardoscalco) [reply LICENSE]: Yep, thanks!

**Comment by riccardoscalco** [comment]
@rpadovani Have you finished the code review?

**Comment by riccardoscalco** [comment]
Thanks!

**Pull request merged**